src/SUMMARY.md

```diff
@@ -39,6 +39,7 @@
 - [Gas Overview](./zksync-specifics/gas.md)
 - [Paymaster Overview](./zksync-specifics/paymaster-overview.md)
 - [Examples](./zksync-specifics/examples/README.md)
+  - [General Flow Paymaster](./zksync-specifics/examples/general-paymaster.md)
```

Comment:
```suggestion
  - [Paymaster General Flow](./zksync-specifics/examples/general-paymaster.md)
```

`General Flow Paymaster` sounds grammatically weird to me but I may be missing some context on where this comes from.

Reply:
"General Flow" would be the adjective in this case

Reply:
Yes, in the docs is called that way